samples/quickstart.js

```diff
@@ -15,7 +15,7 @@
 
 'use strict';
 
-// [START pubsub_quickstart_create_topic]
```

Comment:
Is renaming this region tag to the standard `quickstart` okay? CC: @sduskis, @JustinBeckwith, I wasn't sure how load bearing the region tag is, e.g., will this mean that we break samples on `cloud.google.com`?

Reply:
That is very likely to break samples.  Can you follow up with @jmdobry or @fhinkel on the right way to change this?